packages/mobile/src/screens/edit-track-screen/fields/ReleaseDateField.tsx

```diff
@@ -14,7 +14,7 @@ export const ReleaseDateField = (props) => {
 
   return (
     <ContextualMenu
-      menuScreenName='ReleaseDate'
+      menuScreenName={messages.label}
```

Comment:
technically idk if this should be a messages because we eventually would i18n everything in messages and this might break?